playbook/app/pb_kits/playbook/pb_advanced_table/docs/_advanced_table_selectable_rows_actions_rails.html.erb

```diff
@@ -0,0 +1,122 @@
+<% column_definitions = [
+  {
+    accessor: "year",
+    label: "Year",
+    cellAccessors: ["quarter", "month", "day"],
+  },
+  {
+    accessor: "newEnrollments",
+    label: "New Enrollments",
+  },
+  {
+    accessor: "scheduledMeetings",
+    label: "Scheduled Meetings",
+  },
+  {
+    accessor: "attendanceRate",
+    label: "Attendance Rate",
+  },
+  {
+    accessor: "completedClasses",
+    label: "Completed Classes",
+  },
+  {
+    accessor: "classCompletionRate",
+    label: "Class Completion Rate",
+  },
+  {
+    accessor: "graduatedStudents",
+    label: "Graduated Students",
+  }
+]
+
+actions = [
+  pb_rails("circle_icon_button", props: {
+    icon: "file-csv",
+    variant: "link",
+    id: "export-selected-rows-btn",
+    data: {
+      action_type: "export"
+    }
+  }),
+  pb_rails("circle_icon_button", props: {
+    icon: "trash-alt",
+    variant: "link",
+    id: "delete-selected-rows-btn",
+    data: {
+      action_type: "delete"
+    }
+  })
+]
+%>
+
+<%= pb_rails("advanced_table", props: {
+  id: "selectable_rows_with_actions",
+  table_data: @table_data_no_subrows,
+  column_definitions: column_definitions,
+  selectable_rows: true,
+  enable_toggle_expansion: "none",
+  actions: actions,
+  show_actions_bar: true
+}) %>
+
+<script>
+  window.handleActionClick = function(actionType) {
+    const tableContainer = document.getElementById('selectable_rows_with_actions');
+    if (!tableContainer) return;
+
+    const checkboxes = tableContainer.querySelectorAll('input[type="checkbox"]:checked');
+
+    const selectedCheckboxes = Array.from(checkboxes).filter(checkbox =>
+      checkbox.id !== 'select-all-rows' &&
+      !checkbox.closest('#select-all-rows')
+    );
+
+    const selectedRowIds = selectedCheckboxes.map(checkbox => checkbox.id);
+
+    if (selectedRowIds.length === 0) {
+      alert('No Selection Made');
+    } else {
+      if (actionType === 'export') {
+        alert(`Row ids ${selectedRowIds.join(', ')} will be exported!`);
+      } else if (actionType === 'delete') {
+        alert(`Row ids ${selectedRowIds.join(', ')} will be deleted!`);
+      }
+    }
+  };
+
```

Comment:
Here, I don't think we should need to query select all the checkboxes in this way, we are creating a data attribute under the hood called data-selected-rows that updates dynamically with the selected rows. I would hook into that here instead. We could also export a customevent from the kit in a similar way to [this](https://github.com/powerhome/playbook/pull/4604/files#diff-c75b653efe4a0689bc267ff730378e308a1f8e71fead06e2d7ef612078d9c2eeR152-R155) for easier access